### ISSUE TITLE
Unable to use with electron web pack.

### ISSUE THREAD

**Issue by saikksub** [redxtech/vue-plyr#47]
I am using VueJs-plyr with electron-vue template using `npm`. When I try to register vue-plyr, i get following error:
```
/Users/kksai/ClientW…lyr/dist/plyr.css:1 Uncaught SyntaxError: Invalid or unexpected token
    at createScript (vm.js:80)
    at Object.runInThisContext (vm.js:139)
    at Module._compile (module.js:606)
    at Object.Module._extensions..js (module.js:653)
    at Module.load (module.js:561)
    at tryModuleLoad (module.js:504)
    at Function.Module._load (module.js:496)
    at Module.require (module.js:586)
    at require (internal/module.js:11)
    at Object.<anonymous> (/Users/kksai/ClientW…t/vue-plyr.cjs.js:8)
```

Can you help to find out what is the problem?

**Comment by iCrawl** [comment]
Duplicate of #43 (maybe I should really rename the issue)

**Comment by redxtech** [comment]
Closing as dupe of #43 .

@iCrawl sorry for the wait on that one, I've been busy in real life. We should be seeing a fix soon.
